rules-tests/Php80/Rector/Class_/ClassPropertyAssignToConstructorPromotionRector/FixtureRenamingDisabled/skip_if_different_names.php.inc

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Tests\Php80\Rector\Class_\ClassPropertyAssignToConstructorPromotionRector\Fixture;
+
+use Rector\Tests\Php80\Rector\Class_\ClassPropertyAssignToConstructorPromotionRector\Source\Fixer;
+use Rector\Tests\Php80\Rector\Class_\ClassPropertyAssignToConstructorPromotionRector\Source\SkipParentPublic as BaseSkipParentPublic;
+
+final class SkipWhenNameIsDifferent
+{
+    private Fixer $fixer;
+
+    public function __construct(Fixer $notSameName)
```

Comment:
could you add 1 more fixture with same name property and param that actually transform to property promotion?

Reply:
Done!